api-extractor/src/definitions/ApiProperty.ts

```diff
@@ -15,8 +15,15 @@ class ApiProperty extends ApiMember {
     this.kind = ApiItemKind.Property;
 
     const declaration: any = options.declaration as any; /* tslint:disable-line:no-any */
-    if (declaration && declaration.type) {
-      this.type = declaration.type.getText();
+    if (declaration) {
+      if (declaration.type) {
+        this.type = declaration.type.getText();
+      } else {
+        this.hasIncompleteTypes = true;
+        this.type = 'any';
+      }
+    } else {
+      new Error('No declaration provided');
```

Comment:
This code doesn't throw anything.  Also, is it needed?  Does it occur in any of your unit tests?